### DISCUSSION TITLE
website (gh-pages)

### DISCUSSION THREAD

**Discussion by ghost** [blackjack4494/youtube-dlc#158] [Ideas] [answered]
i am interested in making a new design, restructuring navigation, and all that,
just wanted to ask first, if there are any other specific plans

**Comment by blackjack4494** [comment, marked as answer]
Sure feel free to do that. I just had no time to deal with that also my frontend/ui skills are kinda limited to using frameworks. Some nice clean and simple (minimalistic) design, at least for the landing page would, be dope.  
What are your thoughts? Do you have any specific design ideas already?
i18n would be nice too since it's easier for some to read in their native language.

**Reply by ghost** [reply]
no, nothing yet, gonna start sketching in figma around weekend, i'll share the link once it will turn into something worth discussing further

**Comment by ghost** [comment]
kinda long wait, but i did finally found some time to work on this,
here's a banner (kinda), gonna try to design something in that key

![composed](https://user-images.githubusercontent.com/59403389/96911121-b17cc880-14a0-11eb-9706-3338da47887a.jpg)
